docs/build/quick_start.md

```diff
@@ -0,0 +1,125 @@
+# Quick Start for Developers
```

Comment:
what do you mean for developers? this whole documentation is for developers

Reply:
No, there is quick start for consumers. Not all docs are for developers